src/configurations/destinations/mp/ui-config.json

```diff
@@ -487,45 +568,41 @@
         },
         {
           "type": "dynamicCustomForm",
```

Comment:
@saikumarrs can you check if consent template is all good here ?
This is the first time a destination UI is being migrated from old to new form builder post consent management changes I think

Reply:
LGTM, but please compare it with any existing destinations to be double sure.